sdk/core/Azure.Core/src/Operation.cs

```diff
@@ -16,6 +18,26 @@ namespace Azure
     public abstract class Operation
 #pragma warning restore AZC0012 // Avoid single word type names
     {
+        /// <summary>
+        /// Rehydrates an operation from a <see cref="RehydrationToken"/>.
+        /// </summary>
+        /// <param name="pipeline">The Http pipeline.</param>
+        /// <param name="rehydrationToken">The rehydration token.</param>
+        /// <param name="options">The client options.</param>
+        /// <returns>The long-running operation.</returns>
+        public static Operation<T> Rehydrate<T>(HttpPipeline pipeline, RehydrationToken? rehydrationToken, ClientOptions? options = null) where T : IPersistableModel<T>
```

Comment:
Curious: what happens if this is called with rehydrationToken = `null`?  Can it function correctly?  Should that parameter be nullable?

Reply:
removed nullable